### ISSUE TITLE
Nextcloud beta error

### ISSUE THREAD

**Issue by dsolutiontech** [nextcloud/android#7823]
### Steps to reproduce
1. 
2. 
3. 

### Expected behaviour
- Tell us what should happen

### Actual behaviour
- Tell us what happens

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by AndyScherzinger** [comment]
@dsolutiontech can you please provide some information / details about what is actually not working? 😊

**Comment by dsolutiontech** [comment]
I get this error whenever I try to add a folder via the mobile app from my galaxy note 9

************ CAUSE OF ERROR ************

java.lang.IllegalStateException: The specified child already has a parent. You must call removeView() on the child's parent first.
at android.view.ViewGroup.addViewInner( ViewGroup.java:5958<http://ViewGroup.java:5958>)
at android.view.ViewGroup.addView( ViewGroup.java:5777<http://ViewGroup.java:5777>)
at android.view.ViewGroup.addView( ViewGroup.java:5717<http://ViewGroup.java:5717>)
at android.view.ViewGroup.addView( ViewGroup.java:5690<http://ViewGroup.java:5690>)
at com.owncloud.android<http://com.owncloud.android>.ui.fragment.OCFileListBottomSheetDialog.onCreate( OCFileListBottomSheetDialog.java:131<http://OCFileListBottomSheetDialog.java:131>)
at android.app<http://android.app>.Dialog.dispatchOnCreate( Dialog.java:702<http://Dialog.java:702>)
at android.app.Dialog.show<http://android.app.Dialog.show>( Dialog.java:424<http://Dialog.java:424>)
at com.owncloud.android<http://com.owncloud.android>.ui.fragment.OCFileListFragment.lambda$registerFabListener$2$OCFileListFragment( OCFileListFragment.java:442<http://OCFileListFragment.java:442>)
at com.owncloud.android<http://com.owncloud.android>.ui.fragment.-$$Lambda$OCFileListFragment$aYkoWPoxzAOahKL5AVjT06NhcnM.onClick(Unknown Source:4)
at android.view.View.performClick( View.java:7862<http://View.java:7862>)
at android.view.View.performClickInternal( View.java:7831<http://View.java:7831>)
at android.view.View.access$3600( View.java:879<http://View.java:879>)
at android.view.View$ PerformClick.run<http://PerformClick.run>( View.java:29359<http://View.java:29359>)
at android.os.Handler.handleCallback( Handler.java:883<http://Handler.java:883>)
at android.os.Handler.dispatchMessage( Handler.java:100<http://Handler.java:100>)
at android.os.Looper.loop( Looper.java:237<http://Looper.java:237>)
at android.app<http://android.app>.ActivityThread.main( ActivityThread.java:8167<http://ActivityThread.java:8167>)
at java.lang.reflect.Method.invoke(Native Method)
at com.android<http://com.android>.internal.os.RuntimeInit$ MethodAndArgsCaller.run<http://MethodAndArgsCaller.run>( RuntimeInit.java:496<http://RuntimeInit.java:496>)
at com.android<http://com.android>.internal.os.ZygoteInit.main( ZygoteInit.java:1100<http://ZygoteInit.java:1100>)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30140290
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: samsung
Device: crownqltechn
Model: SM-N9600
Id: QP1A.190711.020
Product: crownqltezh

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: N9600ZHU5FTLA

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by tobiasKaminsky** [comment]
Will be fixed via #7829 and a new 3.14.3 will be released today.
